sway/commands/force_display_urgency_hint.c

```diff
@@ -13,7 +13,7 @@ struct cmd_results *cmd_force_display_urgency_hint(int argc, char **argv) {
 	if (*err) {
 		if (strcmp(err, "ms") != 0) {
 			return cmd_results_new(CMD_INVALID,
-					"Expected 'force_display_urgency_hint <timeout> ms'");
```

Comment:
Hmm, why replace the space with a `|`?

Reply:
I thought it should be consistent with the manual, I documented the command as `force_display_urgency_hint <timeout>|ms` and having the command return a different desired syntax seemed confusing to me. The i3 [userguide](https://github.com/i3/i3/blob/next/docs/userguide#L1176) documents `force_display_urgency_hint` as `force_display_urgency_hint <timeout> ms` since `force_display_urgency_hint ms` is not valid syntax (but is in sway, maybe the implementation should be changed?). I also just noticed that the `force_display_urgency_hint` command is config only on i3 while it can be used over ipc with sway.

Reply:
>maybe the implementation should be changed?

Hmm, yeah, Sway's implementation is buggy. `force_display_urgency_hint ms` doesn't make sense.

Reply:
Atm it just sets `force_display_urgency_hint` to zero. I could create a follow up pull request that changes the syntax to `force_display_urgency_hint <timeout> ms`. Or maybe `force_display_urgency_hint <timeout> [ms]`? I am not sure.

Reply:
> `force_display_urgency_hint <timeout> [ms]`

Yeah this sounds like the correct one. See the i3 tests: https://github.com/i3/i3/blob/ae757c6848b49d7a0423973a39791ba6eca29308/testcases/t/201-config-parser.t#L400

Reply:
Ahh yes, I see. Should I create a separate pull request and add the documentation in this one?

Reply:
If you're willing to change the code to fix it, feel free to edit this PR.

Reply:
Alright!